docs/documentation/install/requirements.md

```diff
@@ -63,14 +63,14 @@ If it says another number such as `0.12` or `5.x.x`, you need to download and in
 
```

Comment:
missed a couple of references to 6 in lines 58 and 60

Reply:
no problem, will have a look later today

Reply:
done